src/js/providers/utils/quality-labels.ts

```diff
@@ -100,7 +114,7 @@ function checkForLevelDuplicates(levels, dupKeys) {
             return false;
         }
         const foundDuplicate = this[key];
```

Comment:
we could remove the `undefined` return value here and have cleaner types by changing this line to `const foundDuplicate = this[key] || false;`